ols/app/models/config.py

```diff
@@ -1050,14 +1050,27 @@ def __eq__(self, other: object) -> bool:
 
 
 class UserDataCollectorConfig(BaseModel):
-    """User data collection configuration."""
+    """User data collection configuration.
+
+    All time information is given in seconds.
+
+    """
 
     data_storage: Optional[DirectoryPath] = None
     log_level: int = logging.INFO
     collection_interval: int = 2 * 60 * 60  # 2 hours
     run_without_initial_wait: bool = False
     ingress_env: Literal["stage", "prod"] = "prod"
     cp_offline_token: Optional[str] = None
+    initial_wait: int = 60 * 5  # 5 minutes in seconds
+    ingress_timeout: int = 30
+    ingress_base_delay: int = 60  # exponential backoff parameter
+    ingress_max_retries: int = 3  # exponential backoff parameter
+    access_token_generation_timeout: int = 5
+    user_data_max_size: int = 100 * 1024 * 1024  # 100 MiB
+    user_agent: str = (
+        "openshift-lightspeed-operator/user-data-collection cluster/{cluster_id}"
```

Comment:
So the implementation expects there is a placeholder for `{cluster_id}` in this config value.
There should be validation in the config for this.

Reply:
done.